karaf/features/src/main/feature/feature.xml

```diff
@@ -279,76 +279,16 @@
-        <feature>brooklyn-software-base</feature>
-    </feature>
-
-    <feature name="brooklyn-software-cm" version="${project.version}" description="Configuration Management modules">
-        <bundle>mvn:org.apache.brooklyn/brooklyn-software-cm-salt/${project.version}</bundle>
-        <bundle>mvn:org.apache.brooklyn/brooklyn-software-cm-ansible/${project.version}</bundle>
-    </feature>
-
-    <feature name="brooklyn-software-osgi" version="${project.version}" description="Brooklyn OSGi Software Entities">
-        <bundle>mvn:org.apache.brooklyn/brooklyn-software-osgi/${project.version}</bundle>
-        <feature>brooklyn-core</feature>
-        <feature>brooklyn-api</feature>
-        <feature>brooklyn-software-base</feature>
-        <bundle dependency="true">mvn:com.google.guava/guava/${guava.version}</bundle>
-    </feature>
-
-    <feature name="brooklyn-software-database" version="${project.version}" description="Brooklyn Database Software Entities">
-        <bundle>mvn:org.apache.brooklyn/brooklyn-software-database/${project.version}</bundle>
-        <feature>brooklyn-core</feature>
-        <feature>brooklyn-api</feature>
-        <feature>brooklyn-software-base</feature>
-        <bundle dependency="true">mvn:com.google.guava/guava/${guava.version}</bundle>
-    </feature>
-
-    <feature name="brooklyn-software-webapp" version="${project.version}" description="Brooklyn Web App Software Entities">
-        <bundle>mvn:org.apache.brooklyn/brooklyn-software-webapp/${project.version}</bundle>
-        <feature>brooklyn-software-base</feature>
-        <bundle dependency="true">wrap:mvn:org.hibernate/jtidy/${jtidy.version}</bundle>
-    </feature>
-
-    <feature name="brooklyn-software-messaging" version="${project.version}" description="Brooklyn Messaging Software Entities">
-        <bundle>mvn:org.apache.brooklyn/brooklyn-software-messaging/${project.version}</bundle>
-        <feature>brooklyn-software-base</feature>
-    </feature>
-
-    <feature name="brooklyn-software-nosql" version="${project.version}" description="Brooklyn NoSQL Data Store Software Entities">
-        <bundle>mvn:org.apache.brooklyn/brooklyn-software-nosql/${project.version}</bundle>
-        <feature>brooklyn-software-base</feature>
-        <feature>brooklyn-software-database</feature>
-        <feature>brooklyn-software-webapp</feature>
-        <bundle dependency="true">mvn:org.mongodb/mongo-java-driver/3.0.3</bundle>
-    </feature>
-
-    <feature name="brooklyn-software-monitoring" version="${project.version}" description="Brooklyn Monitoring Software Entities">
-        <bundle>mvn:org.apache.brooklyn/brooklyn-software-monitoring/${project.version}</bundle>
-        <feature>brooklyn-core</feature>
-        <feature>brooklyn-api</feature>
-        <feature>brooklyn-software-base</feature>
-        <bundle dependency="true">mvn:com.google.guava/guava/${guava.version}</bundle>
-    </feature>
-
```

Comment:
Let's leave these features in, just don't include them in the boot bundles (only change `brooklyn-software-all`). This should be enough not to depend on `brooklyn-library` at build time.